x-pack/legacy/plugins/siem/public/pages/detection_engine/components/signals/index.tsx

```diff
@@ -129,7 +129,9 @@ const SignalsTableComponent: React.FC<SignalsTableComponentProps> = ({
         dataProviders: [],
         indexPattern: indexPatterns,
         browserFields,
-        filters: globalFilters,
+        filters: isEmpty(defaultFilters)
+          ? globalFilters
+          : [...(defaultFilters ?? []), ...globalFilters],
```

Comment:
nit: Doesn't isEmpty catch the `null` and `undefined`, do you need the `?? []`

Reply:
yes but typescript is still yelling at me